packages/nimble-components/src/accordion/specs/IxD.md

```diff
@@ -0,0 +1,108 @@
+# Nimble Accordion (IxD)
```

Comment:
@atmgrifter00 moved to comment thread:

Since we are mentioning the term "disclosure" frequently, it would be good to get some clarity on what we feel are the complete set of differences between an accordion and a disclosure component. One difference is captured already in this document, in that an accordion can have several collapsible sections (particularly indented at the same visual level), whereas a disclosure only has a single collapsible section. However, it's important to note that an accordion can have a single collapsible section, making that particular use-case fairly analogous to a disclosure.

Another possible difference, which could be more convention than anything else, is that disclosure's are typically described and visually represented as a "button". From ARIA's disclosure pattern description:

_" When the controlled content is hidden, the button is often styled as a typical push button with a right-pointing arrow or triangle to hint that activating the button will display additional content."_.

So, I think my questions are summarized with:
- If we wanted Nimble to have an actual "disclosure" component would we want to adopt this visual representation of using a button?
- Is there a good reason for users to not use an accordion with a single collapsible section?

Reply:
- Maybe. This looks like what FAST does. The ARIA pattern has examples of disclosures that look like push buttons, <details> tags, menu buttons, and menu bars. It seems to be some kind of a catch all for when you want a component that looks like another component but without all the complex implementation built into it. So maybe it isn't a component.
- No, I guess not.